translate.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+import urllib.request
+import json
+
+
+class Papago:
+    """
+    Papago 번역은 Papago의 인공 신경망 기반기계 번역 기술(NMT, Neural Machine Translation)로
+    텍스트를 번역한 결과를 반환하는 RESTful API입니다.
+    stack_information.json파일 부분에서 title과 description부분을 번역해준다.
+    """
+
+    def __init__(self, client_id, client_secret, file_name):
+        """
+        client_id, client_secret, api url 설정.
+        file_name을 받음.
+        """
+        self.client_id = client_id
+        self.client_secret = client_secret
+        self.url = "https://openapi.naver.com/v1/papago/n2mt"
+        self.file_name = file_name
+
+    def translate_txt(self, source, target):
+        """
+        title부분과 description부분을 모두 수정해준 뒤
+        기존 파일에 덮어 씌운다.
+        """
+        self.source = source
+        self.target = target
+
+        with open(f"./{self.file_name}", "r") as f:
+            json_data = json.load(f)
+
+        info_dict = json_data["data"]["tools"]["edges"][0]

```

Comment:
pr에서 merge 시 `stack_information.json` 파일에서 conflict 가 있었는데, 제가 커밋으로 해당 파일 구조를 조금 바꿨습니다.
그래서 여기 json_data 파싱할때 
```suggestion
        info_dict = json_data[0]
```
이렇게만 해주면 될거 같아요